build/tests.mk

```diff
@@ -222,6 +255,18 @@ check-crypto-debug:
 	$(call zencode-tests,${test-exec})
 	cat /tmp/zenroom-test-summary.txt
 
+check-cortex-m: test-exec := qemu-system-arm -M mps2-an385 -kernel src/zenroom.bin -semihosting -nographic
```

Comment:
`qemu-system-arm` appears in various parts. You can split it into a dedicated script and then use `source`.

Reply:
Instead of invoking the qemu with predefined arguments, use a wrapper script for abstraction.